integration-libs/cpq-quote/README.md

```diff
@@ -0,0 +1,7 @@
+This library provides Discount percentage to Spartacus UI.
+
+As an end user utilizing the SAP Commerce B2B Storefront,
```

Comment:
I think we can format this file better:

This library provides a discount percentage to the Spartacus UI.

As an end user utilizing the SAP Commerce B2B Storefront, it's imperative for transparency and informed decision-making that users can easily access and comprehend the discount percentage displayed on the Quote Table.

Additionally, maintaining the UI consistency with the current Accelerator UI for the proposal document enhances user familiarity and usability. This streamlines the overall experience and minimizes potential confusion or disruptions in workflow.